lib/new_relic/agent/instrumentation/elasticsearch/instrumentation.rb

```diff
@@ -10,7 +10,11 @@ module Elasticsearch
     OPERATION = 'perform_request'
     INSTRUMENTATION_NAME = NewRelic::Agent.base_name(name)
 
-    def perform_request_with_tracing(method, path, params = {}, body = nil, headers = nil)
+    # We need the positional arguments `params` and `body`
+    # to capture the nosql statement
+    # *args protects the instrumented method if new arguments are added to
+    # perform_request
+    def perform_request_with_tracing(method, path, params = {}, body = nil, headers = nil, *args)
```

Comment:
Would this be an appropriate time to do `_*args` or can it be inferered we're not doing anything with the additional arguments

Reply:
Great call. But `*_args`.

Reply:
Good idea! 

`_*args` isn't valid Ruby syntax, but `*_args` is. The `*` is a wildcard, and then the part that follows is the string/name of the argument.  

I've updated all the arguments in this method to have `_`s prefixed if we're not using them in 4a44b9c5b.